R/parquet.R

```diff
@@ -73,9 +96,26 @@ pl$read_parquet = function(
     row_count_name = NULL,
     row_count_offset = 0L,
     low_memory = FALSE) {
+
+  #construct a derived call
   mc = match.call()
```

Comment:
@etiennebacher  Interesting using quote + eval instead `...`, I have not thought of that. What would be the pros and cons?

Reply:
I agree scan_parquet needs a bigger overhaul. I would try to do that in separate PR.

Reply:
This quote + eval() was already done in `$read_csv()`, so I just took it there but I'm not the one who implemented it. I can't think of cons for now

Reply:
@eitsupi what do you think pros and cons of using ... vs match/eval ?

Reply:
I don't really understand the intent of your question, but I think `match.call` is more convenient here since using `...` as a function argument has the disadvantage that auto-complete cannot be used.

Reply:
sry my bad, I blame the fever :)